packages/react-components/react-toolbar/src/stories/ToolbarWithPopover.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Toolbar, ToolbarProps } from '../index';
+import { ToolbarButton } from '../ToolbarButton';
+import { ToolbarDivider } from '../ToolbarDivider';
+import { ToolbarToggleButton } from '../ToolbarToggleButton';
+import { Popover, PopoverTrigger, PopoverSurface } from '@fluentui/react-popover';
+import { CalendarMonthRegular } from '@fluentui/react-icons';
+
+export const WithPopover = (props: Partial<ToolbarProps>) => (
+  <Toolbar {...props} size="small">
+    <Popover>
+      <PopoverTrigger>
+        <ToolbarButton>Click here</ToolbarButton>
+      </PopoverTrigger>
+      <PopoverSurface>
+        <div>Popover content</div>
+      </PopoverSurface>
+    </Popover>
+    <ToolbarDivider />
+    <Popover>
+      <PopoverTrigger>
+        <ToolbarToggleButton icon={<CalendarMonthRegular />} />
```

Comment:
I think we will need to have something more complex to solve this case.
In v0, the icon in the button needs to be filled while popup is open, let's check with design, but I think this is still a valid requirement.

Using toggle however does not enforce the state of the toggle to be linked with the state of the Popover - you can open it by clicking, but if you close it by clicking outside or ESC, the button stays pressed.

Also, aria-pressed is not a good state to announce when a popover is open. It might be aria-expanded, but even in that case I would vote more for not having any aria state there as the additional narration does not bring relevant information to the user.

Reply:
Removed the `ToogleButton` from popover stories and tooltip